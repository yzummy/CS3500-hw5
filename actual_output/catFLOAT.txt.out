
___Entering new scope...

TOKEN: CAT 		 LEXEME: cat
TOKEN: LPAREN 		 LEXEME: (
TOKEN: FLOATCONST 		 LEXEME: -27.834
CONST -> FLOATCONST
FACTOR -> CONST
TOKEN: RPAREN 		 LEXEME: )
MULT_OP_LIST -> epsilon
TERM -> FACTOR MULT_OP_LIST
HERE 2222
ADD_OP_LIST -> epsilon
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC
EXPR -> ARITHLOGIC_EXPR
OUTPUT_EXPR -> CAT ( EXPR )
-27.83
EXPR -> OUTPUT_EXPR
START -> EXPR

---- Completed parsing ----

Value of the expression is: NULL

___Exiting scope...

