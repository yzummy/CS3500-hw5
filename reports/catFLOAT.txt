							      >	-27.83

---- Completed parsing ----					---- Completed parsing ----

							      >	Value of the expression is: NULL
