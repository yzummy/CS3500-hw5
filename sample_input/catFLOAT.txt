cat(-27.834)
