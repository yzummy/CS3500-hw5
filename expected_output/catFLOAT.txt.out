-27.83

---- Completed parsing ----

Value of the expression is: NULL
